lib/shelley/src/Cardano/Wallet/Shelley/Network/Blockfrost.hs

```diff
@@ -465,20 +934,34 @@ instance FromBlockfrost BF.ProtocolParams ProtocolParameters where
             , ..
             }
 
+instance FromBlockfrost BF.TxHash (Hash "Tx") where
+    fromBlockfrost txHash =
+        let hash = BF.unTxHash txHash
+        in first (InvalidTxHash hash) $ fromText hash
```

Comment:
(Style:

```suggestion
    fromBlockfrost txHash = first (InvalidTxHash hash) $ fromText hash
      where hash = BF.unTxHash txHash
```
For consistency?)

Reply:
I chose `let` because thought that according to our style there must be a line feed after `where` so it makes it occupy 3 lines while `let` occupies only 2 LOC.